### PR TITLE
Update privacy statement: Change president

### PR DESCRIPTION
**Pull request by larswmh**
This PR changes the president mentioned in the privacy statement.

image
image

---
Internal Tracking ID: [EXPOSUREAPP-14634](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14634)

### CODE REVIEW

File: src/data/privacy_de.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
                 "Willkommen auf der offiziellen Website der Corona-Warn-App der Bundesregierung.",
                 "<b>1. Verantwortlicher und Datenschutzbeauftragter</b>",
                 "Verantwortlich für die Verarbeitung Ihrer personenbezogenen Daten gemäß den Vorschriften der Datenschutzgrundverordnung (DSGVO) und des Bundesdatenschutzgesetzes (BDSG) bei Besuch dieser Website ist:",
-                "Robert Koch-Institut (RKI)<br />Nordufer 20<br />13353 Berlin<br />vertreten durch den Präsidenten Prof. Dr. L. H. Wieler",
+                "Robert Koch-Institut (RKI)<br />Nordufer 20<br />13353 Berlin<br />vertreten durch den Präsidenten i.V. Prof. Dr. Schaade",
```
Comment (MikeMcC399) [inline src/data/privacy_de.json:18]: written out this means:

"vertreten durch den Präsidenten in Vertretung Prof. Dr. Schaade"

which sounds a bit strange!

Does this mean that Prof. Dr. Schaade is not the real President and he is only standing in for him?

Comment (MikeMcC399) [reply src/data/privacy_de.json:18]: @larswmh 

Yes, thanks for asking. I have hidden my comment now as "resolved".

**Review by dsarkar — APPROVED** [review]

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
This is not approved. You'll receive a mail from me shortly!

**Review by brianebeling — APPROVED** [review]

**Comment by Ein-Tim** [comment]
Mail's out, latest info I got was to use "vertreten durch den Präsidenten" bzw. "vertreten durch den komm. Präsidenten", as it is used in the Impressum of rki.de

**Comment by dsarkar** [comment]
changes approved.

**Comment by dsarkar** [comment]
@Ein-Tim Many thanks!

**Comment by larswmh** [comment]
We changed it to "Vertreten durch den Präsidenten" / "Represented by the President", which is exactly how it is used on the RKI website

**Review by Ein-Tim — APPROVED** [review]
Looks good to me now, thanks!

**Pull request merged**